src/base/orgs/View/Projects.svelte

```diff
@@ -62,11 +62,11 @@
   {#await seed.getProjects(profile?.id)}
```

Comment:
We have the option to return `Project` instances from this function, similar to the `Org.getMulti` function.
I did not implement it, under the reason that we would be fetching and doing a lot of overhead to get just `projectInfo` and we can leave that for when a user visits the project.